cmd/copilot/description.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
 // +build !windows
```

Comment:
Maybe we leave it for now just in case?
```
"As of this release, the new syntax is fully supported, and all Go files should be updated to have both forms with the same meaning."
-https://golang.org/doc/go1.17
```

Reply:
qq: does it mean we remove `// +build !windows` now?

Reply:
In theory, yes. https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md
But maybe we do it in a separate PR so it's easier to revert?

Reply:
Agreed. My IDE constantly adds this as well.